routes/api/admin.js

```diff
@@ -1,11 +1,12 @@
 import express from 'express';
+import { checkToken, checkAdmin } from '../../middlewares';
 import userController from '../../controllers/admin';
 import validators from '../../middlewares/validations/validations';
 
 const router = express.Router();
-router.post('/admin/users', validators.validateCreateUser, userController.createUser);
-router.get('/admin/users', userController.getUsers);
-router.patch('/admin/users/:userId', userController.updateUser);
-router.delete('/admin/users/:userId', userController.deleteUser);
+router.post('/admin/users', validators.validateCreateUser, checkToken, checkAdmin, userController.createUser);
+router.get('/admin/users', checkToken, checkAdmin, userController.getUsers);
+router.patch('/admin/users/:userId', checkToken, checkAdmin, userController.updateUser);
```

Comment:
Consider using `PUT` instead of `PATCH`. Read [this article](https://medium.com/backticks-tildes/restful-api-design-put-vs-patch-4a061aa3ed0b) for a detailed comparison of `PUT` and `PATCH`